src/rulesets/oas/functions/__tests__/oasPathParam.test.ts

```diff
@@ -35,8 +35,7 @@ describe('oasPathParam', () => {
     expect(results).toEqual([
       {
         code: 'path-params',
-        message:
-          'The path "**/foo/{bar}**" uses a parameter "**{bar}**" that does not have a corresponding definition.\n\nTo fix, add a path parameter with the name "**bar**".',
+        message: 'The path `/foo/{bar}` uses a parameter `{bar}` that does not have a corresponding definition.',
```

Comment:
@philsturgeon Do we want quotes? I know we mix single with double quotes, as well as we have grave accents in some places, but I think double quotes are the most popular?

Reply:
Double quotes for a trying and maybe these backticks for dynamic? Definitely beats the stars and stuff.